webapi/src/main/java/org/sakaiproject/webapi/controllers/CardGameController.java

```diff
@@ -75,16 +82,32 @@ public class CardGameController extends AbstractSakaiApiController {
     @Autowired
     private PrivacyManager privacyManager;
 
+    @Autowired
+    private ProfileImageLogic profileImageLogic;
+
+    private int minAttempts;
+    private double minHitRatio;
+    private boolean showOfficialPhoto;
+    private boolean skipNoImageUsers;
+
+
+    @PostConstruct
+    public void init() {
+        minAttempts = serverConfigurationService.getInt("cardgame.minAttempts", MIN_ATTEMPTS_DEFAULT);
+        minHitRatio = serverConfigurationService.getDouble("cardgame.minHitRatio", MIN_HIT_RATIO_DEFAULT);
+        showOfficialPhoto = serverConfigurationService.getBoolean("cardgame.showOfficialPhoto", SHOW_OFFICIAL_PHOTO_DEFAULT);
+        skipNoImageUsers = serverConfigurationService.getBoolean("cardgame.skipNoImageUsers", SKIP_NO_IMAGE_USERS_DEFAULT);
+    }
+
     @GetMapping(value = "/sites/{siteId}/card-game/config", produces = MediaType.APPLICATION_JSON_VALUE)
     public Map<String, Object> getConfig(@PathVariable String siteId) {
         checkSakaiSession();
         checkSite(siteId);
 
         HashMap<String, Object> config = new HashMap<>();
-        config.put("minAttempts", serverConfigurationService.getInt("cardgame.minAttempts", MIN_ATTEMPTS_DEFAULT));
-        config.put("minHitRatio", serverConfigurationService.getDouble("cardgame.minHitRatio", MIN_HIT_RATIO_DEFAULT));
-        config.put("showOfficialPhoto", serverConfigurationService.getBoolean("cardgame.showOfficialPhoto",
-                SHOW_OFFICIAL_PHOTO_DEFAULT));
+        config.put("minAttempts", minAttempts);
+        config.put("minHitRatio", minHitRatio);
+        config.put("showOfficialPhoto", showOfficialPhoto);
 
```

Comment:
Not sure where this is called but should skipNoImageUser be added here too?

Reply:
No, it's not needed. This map defines the config the frontend needs. The frontend does not need to know about skipNoImageUser for now, the users are filtered on the backend.